pkg/apis/core/v1/defaults.go

```diff
@@ -157,6 +159,29 @@ func SetDefaults_Pod(obj *v1.Pod) {
 				}
 			}
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) &&
```

Comment:
Follow up with a test? :)

Reply:
Yes, a more comprehensive test is warranted, sorry for the miss. I'll bring it with GetPodQoS -> PodQOSClass change.
Having a stiff drink rn :)

Reply:
@thockin Please review PR https://github.com/kubernetes/kubernetes/pull/116684 that adds missing unit test for defaulting